README.md

```diff
@@ -56,6 +56,33 @@ ninja
   env variable for the KeyStore file path.
 * Run `./tools/build-amber-sample.sh [build output directory path]`.
 
+#### Android plain executable
+
+It is possible to obtain a plain executable for Android, as opposed to an APK,
+with the following:
+
+```
+./tools/update_build_version.py . samples/ third_party/
+
+mkdir build
+cd build
+mkdir app
+mkdir libs
+${ANDROID_NDK_HOME}/ndk-build -C ../samples NDK_PROJECT_PATH=. NDK_LIBS_OUT=`pwd`/libs NDK_APP_OUT=`pwd`/app
```

Comment:
Can we add this build line into kokoro/ndk-build/build.sh so we build both the ndk library and this sample app? Then we'll know we don't break it in the future.